scripts/common/kubernetes.sh

```diff
@@ -549,3 +549,31 @@ function kubernetes_pod_completed() {
 
     return 1
 }
+
+function kubernetes_is_current_cluster() {
+    local api_service_address="$1"
+    if cat /opt/replicated/kubeadm.conf 2>/dev/null | grep -q "${api_service_address}"; then
```

Comment:
It's possible they've changed their load balancer address and run the `set-kubeconfig-server` task. How about checking `/etc/kubernetes/kubelet.conf` instead?

Reply:
what about on secondary nodes? ```$ ls /etc/kubernetes/
manifests  pki```

Reply:
ill check both